### ISSUE TITLE
Notary + Pull through cache registry mirrors

### ISSUE THREAD

**Issue by kkroo** [theupdateframework/notary#1373]
Will Notary validate signed images fetched using "registry-mirrors" option on CLI?
I have a use case where I have a signed image pushed to central Hub, and multiple remote clusters running local registries that cache images that are pulled. Can the remote docker daemons use these cached images while maintaining trust from the remote Notary instance the images were pushed to?

**Comment by HuKeping** [comment]
Yes if the name of the cached images are the same with the images you pushed to the central Hub.

Notary will use the `GUN` to identify the trust data.
